api/patientInvites.go

```diff
@@ -91,7 +94,34 @@ func (a *Api) AcceptPatientInvite(res http.ResponseWriter, req *http.Request, va
 			return
 		}
 
-		patient, err := a.createClinicPatient(ctx, *conf)
+		accept := models.AcceptPatientInvite{}
+		if req.ContentLength > 0 {
+			if err := json.NewDecoder(req.Body).Decode(&accept); err != nil {
+				a.sendError(ctx, res, http.StatusInternalServerError, STATUS_ERR_CREATING_PATIENT,
+					fmt.Errorf("error decoding accept patient invite body: %w", err),
```

Comment:
```suggestion
					fmt.Errorf("decoding accept patient invite body: %w", err),
```
nit: it's already an error value, no need to repeat it